src/runner.ts

```diff
@@ -137,11 +141,25 @@ export class Runner {
     return new Project({
       tsConfigFilePath: path.join(process.cwd(), args.p),
       compilerOptions: {
-        strictNullChecks: true,
-        strictPropertyInitialization: true,
-        noImplicitAny: true,
-        noImplicitReturns: true,
+        strictNullChecks: setFlag,
+        strictPropertyInitialization: setFlag,
+        noImplicitAny: setFlag,
+        noImplicitReturns: setFlag,
       },
     });
   }
+
+  /**
+   * Verifies that the project user passes in through CLI compiles initially with flags set to false.
+   * @param {ArgumentOptions} args - CLI Arguments containing project properties.
+   */
+  private verifyProject(args: ArgumentOptions): void {
+    if (
+      !this.parser.parse(this.createProject(args, false)).every(diagnostic => {
+        return diagnostic.getCategory() !== ts.DiagnosticCategory.Error;
+      })
+    ) {
+      throw new Error('Project not current compiling with flags set to false.');
```

Comment:
the with original flags may be half true half false, may be just say "Project not current compiling with original ts config flags"